### PR TITLE
[release-1.23] announce 1.21.6

### PR DESCRIPTION
**Pull request by jaellio**
Security release related to https://github.com/istio/istio.io/pull/15710

### CODE REVIEW

File: content/en/news/releases/1.21.x/announcing-1.21.6/index.md
Comment (dhawton) [inline content/en/news/releases/1.21.x/announcing-1.21.6/index.md:19]:
```suggestion
- **Fixed** `VirtualMachine` `WorkloadEntry` locality label missing during auto registration.
```

**Review by dhawton — COMMENTED** [review]
minor nit, otherwise LGTM

**Review by dhawton — APPROVED** [review]
LGTM if it merges tonight, otherwise let's bump publish date

**Comment by wilsonwu** [comment]
Also need a manual cherry pick to master.

**Comment by dhawton** [comment]
> Also need a manual cherry pick to master.

Yup.. I will be taking care of it once they are merged and the security releases are all handled.

**Review by dhawton — APPROVED** [review]

**Pull request merged**
